data/README.md

```diff
@@ -4,6 +4,8 @@ This directory contains language-specific data files. Most importantly, you will
 
 1. A list of unique characters for the target language (e.g. English) in `data/alphabet.txt`
 2. A binary n-gram language model compiled by `kenlm` in `data/lm/lm.binary`
-3. A trie model compiled by `generate_trie.cpp` in `data/lm/trie`
+3. A trie model compiled by [generate_trie](https://github.com/mozilla/DeepSpeech#using-the-command-line-client) (in native_client) in `data/lm/trie`
 
-For more information on how to create these resources, see `data/lm/README.md`
+
+You can find previous versions for generating trie (and pre-built binaries) from [here](https://github.com/mozilla/DeepSpeech/releases). 
```

Comment:
I'm not sure about that, both the wording and adding a link to releases, especially inside `data/README.md`

Reply:
Got it! fixing.